src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -2303,19 +2304,36 @@ void jit_prelu_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs,
 jit_relu_emitter::jit_relu_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
                                    dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
                                    const std::shared_ptr<ov::Node>& node)
-    : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {}
+    : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+    if (const auto leaky_relu = ov::as_type_ptr<LeakyReluNode>(node)) {
+        alpha = leaky_relu->get_slope();
+    } else if (ov::is_type<ov::op::v0::Relu>(node)) {
+        alpha = 0.f;
+    } else {
+        OV_CPU_JIT_EMITTER_THROW("Can't cast to LeakyReluNode or ReluNode");
+    }
+    prepare_table();
+}
 
 jit_relu_emitter::jit_relu_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
                                    dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                   float alpha,
                                    const ov::element::Type exec_prc)
-    : jit_emitter(host, host_isa, exec_prc) {}
+    : jit_emitter(host, host_isa, exec_prc),
+      alpha(alpha) {
+    prepare_table();
+}
 
 size_t jit_relu_emitter::get_inputs_count() const {
     return 1;
 }
 
 size_t jit_relu_emitter::get_aux_vecs_count() const {
-    return 1;
+    return (alpha != 0.0f) ? 2 : 1;
```

Comment:
Minor:
You can introduce method `is_relu()` which returns `true` if `alpha` is zero. Then you can replace all `(alpha != 0.0f)` with `!is_relu()`. Just to avoid code duplication 😃

Reply:
That would indeed be helpful! I'll make the change.